test/integration/helpers.js

```diff
@@ -114,32 +114,37 @@ module.exports = {
     );
   },
   /**
-   * Invokes the mocha binary for the given fixture using the JSON reporter,
-   * returning the **raw** string output, as well as exit code.
+   * Invokes the mocha binary with the given arguments fixture using
+   * the JSON reporter. Returns the child process and a promise for the
+   * results of running the command. The result includes the **raw**
+   * string output, as well as exit code.
    *
    * By default, `STDERR` is ignored. Pass `{stdio: 'pipe'}` as `opts` if you
-   * want it.
-   * @param {string} fixturePath - Path from __dirname__
+   * want it as part of the ocwresultutput.
```

Comment:
`ocwresultutput`?

Reply:
Looks like I was briefly possessed by a daemon. Fixed, squashed, and pushed